multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -36,6 +36,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("Gabut Scans", "https://gabutscans.com", "id", overrideVersionCode = 1),
         SingleLang("Gecenin Lordu", "https://geceninlordu.com", "tr", overrideVersionCode = 1),
         SingleLang("GoGoManga", "https://gogomanga.fun", "en", overrideVersionCode = 1),
+        SingleLang("Gremory Mangas", "https://gremorymangas.com/", "es"),
```

Comment:
Remove the trailing slash.

```suggestion
        SingleLang("Gremory Mangas", "https://gremorymangas.com", "es"),
```